.github/workflows/update-featured-repo.yml

```
@@ -0,0 +1,49 @@
+name: Update Featured Repository
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)